gokart/build.py

```diff
@@ -28,6 +32,15 @@ class GokartBuildError(Exception):
     pass
 
 
+class HasLockedTaskException(Exception):
+    pass
+
+
+class TaskLockExceptionRaisedFlag:
+    def __init__(self):
+        self.flag: bool = False
```

Comment:
[question] @mski-iksm do we need `TaskLockExceptionRaisedFlag` class? 

looks like a simple bool field is enough ? ( `task_lock_exception_raised = True`

Reply:
This is required, because if `task_lock_exception_raised` was simple bool, we cannot access it from `when_failure()` function.